boards/shields/waveshare_pico_oled_1_3/boards/rpi_pico.overlay

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2023 TOKITA Hiroshi
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&pinctrl {
+	i2c1_default: i2c1_default {
+		waveshare_pico_oled_1_3_group1 {
+			pinmux = <I2C1_SDA_P6>, <I2C1_SCL_P7>;
+			input-enable;
+			input-schmitt-enable;
+		};
+	};
+};
+
+&i2c1 {
```

Comment:
No connector provided, but I guess in specific case of Raspberry Pico (oine single board reference), it is fine. @jfischer-no thoughts ?

Reply:
There is pico_spi.
https://github.com/zephyrproject-rtos/zephyr/blob/8e7c7c62b15cbc09633e48f95eec6aa9150a4791/boards/arm/rpi_pico/rpi_pico-common.dtsi#L151C1-L151C20
I think it make sense to add pico_i2c.

Reply:
It makes sense.
For consistency reasons, I added aliases as pico_spi0, pico_spi1, pico_i2c0, and pico_i2c1.